run-composed

```diff
@@ -0,0 +1,107 @@
+#!/usr/bin/env bash
+# See https://github.com/kabisaict/jenkins-docker for more information.
+#
+# Usage:
+# run-composed [CI command]
+#
+# Examples:
+# run-composed 'bin/ci'
+# run-composed 'mvn clean install'
+set -o pipefail
+trap cleanupContainer EXIT
+
+function cleanupContainer() {
+  if [ -n "$CID" ]; then
+    CONTAINER_RUNNING=$(docker inspect --format='{{.State.Running}}' $CID)
+    if [ "$CONTAINER_RUNNING" = "true" ]; then
+      echo "Cleaning up docker container"
+
+      # In regular workflows this is done automatically, but when the job is aborted
+      # and the container is killed this cleanup won't happen, so do it here.
+      docker exec $CID chown -R $(id -u `whoami`) /workspace
+      docker kill $CID > /dev/null
+    fi
+  fi
+}
+
+function abort() {
+  echo "$(tput setaf 1)$1$(tput sgr 0)"
+  exit 1
+}
+
+if [ $# -ne 1 ]; then
+  abort "$0 requires exactly one argument (of bash script)."
+fi
+
+# DOCKER_OPTS is now deprecated in favor of DOCKER_RUN_OPTS.
+# If it is set assign it to DOCKER_RUN_OPTS and print a warning.
+if [ -n "$DOCKER_OPTS" ]; then
```

Comment:
Might as well all of this, since this is new code. No need for backwards compatibility :-)